### ISSUE TITLE
Communication Between two machines are not happening

### ISSUE THREAD

**Issue by sumitpaulde** [data-capsule/gdp-router#18]
Hi @KeplerC 

I am trying to establish communication between two ros2 nodes. Both nodes are connected to different access points in different rooms. However the access points are connected with the same subnet.

I tried on machine A 
```
export SGC_CONFIG=talker.toml
export GATEWAY_IP=130.1.1.A
cargo run router

```
and on machine B
```
export SGC_CONFIG=listener.toml
export GATEWAY_IP=130.1.1.A
cargo run router
```

Should we exchange the crypto directory from machine A to B and vice versa or it should be one way?

**Comment by KeplerC** [comment]
Yes you should exchange the crypto directory. The 130.1.1.A should be a machine that also runs a copy of SGC that facilitates connection in the middle. 

If machine A and machine B are able to connect to google (general internet connection), you can also try our webrtc version: https://github.com/data-capsule/fogros2-sgc. This doesn't rely on hardcoding the IP address and switches in the middle. (but still need to exchange the crypto folder)

**Comment by sumitpaulde** [comment]
Hi @KeplerC 
Thanks a lot for your reply.
When I exchange the /crypto directory from Node A to Node B, the crypto directory in Node B gets replaced, and similar happens in Node A.
```
scp -r crypto USER@MACHINE_B_IP_ADDR:/SGC_PATH/scripts/
```
Just wondering if is it the right way, or I should store the crypto directory for each Node in some other location.

I already tried with the fogros2-sgc. Using the router config as well. For the publisher node 
```
export SGC_CONFIG=talker.toml
cargo run router
``` 
and for the subscriber node 
```
export SGC_CONFIG=listener.toml
cargo run router
```
I have not changed any configuration in the toml files, and even I am unable to see the data exchange between the two nodes. I have followed the readme. Am I missing something?


Out of curiosity, I just wanted to know if there is any documentation on how to set up the SGC proxy and SGC Router.
Some examples might be most useful for a newbie like me.
